tests/index.js

```diff
@@ -683,7 +685,7 @@ describe('Thorn', () => {
                     // 4) Do a request with this user and make sure you get a 200.
                 });
 
-                it('should retry clean up on 401\'s', function*() {
+                it('should retry cleanup on 401\'s', function*() {
```

Comment:
These are just wording nitpicks. If everyone agrees I'll split them out to a separate PR.